docs/lessons/next.rst

```diff
@@ -0,0 +1,34 @@
+Next Steps
+----------
+
+These tutorials are only the first steps in using MicroPython with the
+BBC micro:bit. A musical analogy: you've got a basic understanding of
+a very simple instrument and confidently play "Three Blind Mice".
+
+Please don't be disheartened!
```

Comment:
Why not change this to - a positive statement instead - like "This is great!"

Or "... Which means you can do more than most adults!"

Or similar.